src/ORM/FieldType/DBDate.php

```diff
@@ -227,6 +243,11 @@ public function Format($format)
         if (!$this->value) {
             return null;
         }
+
+        $format = preg_replace_callback('/{o(rd(inal)?)?}/i', function ($matches) {
```

Comment:
this seems unduly complex - can we not just use `{o}`?

Reply:
We could do, though I was originally going to just support `{ordinal}`: as we’re extending the syntax, I thought verbosity is probably a good thing as it’s much more obvious at a glance.

Reply:
Then you don't need regular expression replace, just str_replace.

Reply:
Then you don't need regular expression replace, just str_replace.

Reply:
Add the quotes around your string replacement here instead of in getDayOfMonthOrdinal().